app/services/builders/external_project.rb

```diff
@@ -0,0 +1,15 @@
+module Builders
+  class ExternalProject < BaseService
+    def initialize(repository_data)
+      @repository_data = repository_data
+    end
+
+    def call
+      ::ExternalProject.find_or_initialize_by(github_id: @repository_data[:id]) do |project|
```

Comment:
yep, because we do not save it on cases where the projects returned by github API does not have any pull request for the user we are processing, so we just want to find/initialize to check later if is necessary persist it